osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkinConfiguration : DefaultSkinConfiguration
+    {
+        public const double LATEST_VERSION = 2.5;
+
+        /// <summary>
+        /// Legacy version of this skin.
+        /// </summary>
+        public double? LegacyVersion { get; internal set; }
```

Comment:
`double` seems like an unfortunate data type choice for me here, worried about floating-point shenanigans. I guess either `decimal`, `string` or a proper `struct` with major-minor fields would be better choices if I were to suggest something else.

Reply:
I think it's fine to use `decimal` for now as this is a legacy version.